app/models/client.rb

```diff
@@ -648,28 +648,16 @@ def to_jsonapi
       "deleted_at" => deleted_at ? deleted_at.iso8601 : nil,
     }
 
-    Rails.logger.info "SKV - TO_JSONAPI BEGIN: uid, doi_counts, dois_total, dois_current_year, dois_last_year"
-    Rails.logger.info uid
-    Rails.logger.info doi_counts
-    Rails.logger.info dois_total
-    Rails.logger.info dois_current_year
-    Rails.logger.info dois_last_year
-    Rails.logger.info "SKV - TO_JSONAPI MIDDLE: attributes"
-    Rails.logger.info attributes
-    Rails.logger.info "SKV - TO_JSONAPI END"
-
     { "id" => symbol.downcase, "type" => "clients", "attributes" => attributes }
   end
 
-  def self.export(query: nil)
-    Rails.logger.info "SKV - EXPORTING CLIENTS - BEGIN"
-
+  def self.export(query: nil, include_deleted: false)
```

Comment:
This now changes the default behaviour, it was originally to always include deleted, which was what was generally preferred for the export. Is the intention now to make it exclude and only include optionally?

Reply:
Yes, it does change the default behavior.  The intention was to make its behavior consistent with the rest of the API and then change where it is used, which is hopefully only in the Salesforce sync job. If you leave the default behavior, including deleted, then there is no way to affect the behavior with this option.  Currently the option's value has no affect.  I can change it so it pays attention to include_deleted=true and include_deleted=false.

Do we have this documented anywhere?